components/gameboard/DiscardPile.tsx

```diff
@@ -1,13 +1,14 @@
 import styles from "./Piles.module.css";
-import { PileProps } from "./types";
+import { DiscardPileProps } from "./types";
 
-function DiscardPile({ onClick }: PileProps) {
+function DiscardPile({ onClick, card }: DiscardPileProps) {
   return (
     <>
       <section className={styles.container}>
         <h2 className={styles.discardHeadline}>Discard Pile</h2>
         <img
-          src="/cards/blank.png"
+          key={card ? card.value : ""}
```

Comment:
You don't need a `key` here. This is only required if you render arrays.
```suggestion
          
```

Reply:
You're right, my reason for this was that I thought I need the value in order to calculate the round score.
Thinking about it now, I will propably calculate scores in backend, so this isn't needed.
Thanks!